src/Avalonia.Input/InputElement.cs

```diff
@@ -196,6 +196,24 @@ public class InputElement : Interactive, IInputElement
         /// Defines the <see cref="DoubleTapped"/> event.
         /// </summary>
         public static readonly RoutedEvent<TappedEventArgs> DoubleTappedEvent = Gestures.DoubleTappedEvent;
+        
+        /// <summary>
+        /// Defines the <see cref="PointerTouchPadGestureMagnify"/> event.
+        /// </summary>
+        public static readonly RoutedEvent<PointerTouchPadGestureMagnifyEventArgs> PointerTouchPadGestureMagnifyEvent 
+            = Gestures.PointerTouchPadGestureMagnifyEvent;
+        
+        /// <summary>
+        /// Defines the <see cref="PointerTouchPadGestureRotate"/> event.
+        /// </summary>
+        public static readonly RoutedEvent<PointerTouchPadGestureRotateEventArgs> PointerTouchPadGestureRotateEvent 
+            = Gestures.PointerTouchPadGestureRotateEvent;
+        
+        /// <summary>
+        /// Defines the <see cref="PointerTouchPadGestureSwipe"/> event.
+        /// </summary>
+        public static readonly RoutedEvent<PointerTouchPadGestureSwipeEventArgs> PointerTouchPadGestureSwipeEvent 
+            = Gestures.PointerTouchPadGestureSwipeEvent;
```

Comment:
Should I remove this?

Reply:
Yes, please remove it from InputElement.
It still should be possible to use in from xaml like
```
<Canvas Gestures.PointerTouchPadGestureSwipe="" />
```